File: src/main/java/no/nav/pensjon/innsyn/domain/source/HardcodedOpptjening.java
package no.nav.pensjon.innsyn.domain.source;

import no.nav.pensjon.innsyn.domain.Opptjening;
import no.nav.pensjon.innsyn.source.EntityGetter;

import java.util.List;

public class HardcodedOpptjening implements EntityGetter<Opptjening> {

    @Override
    public List<Opptjening> getEntities() {
        return List.of(
                new Opptjening(
                        "type",
                        "status",
                        2019,
                        123456.78d,
                        12.34d,
                        15.75d));
    }
}

File: src/test/kotlin/no/nav/pensjon/innsyn/domain/source/HardcodedOpptjening.kt
package no.nav.pensjon.innsyn.domain.source

import no.nav.pensjon.innsyn.domain.Opptjening

class HardcodedOpptjening : EntityGetter<Opptjening?> {
    override val entities = listOf(
                Opptjening(
                        "type",
                        "status",
                        2019,
                        123456.78,
                        12.34,
                        15.75))
}